### ISSUE TITLE
Feature Request: Allow ignore of checks specific to a shellcheck version

### ISSUE THREAD

**Issue by jpbuecken** [koalaman/shellcheck#2806]
#### For new checks and feature suggestions
- [ x ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ x ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:
Feature request to restrict disabled checks to shellcheck versions.

Usually you follow:
https://www.shellcheck.net/wiki/Ignore
and the header of the shellcheck looks like:
```sh

#!/bin/bash
# shellcheck disable=SC2317

```

My idea is to add a version parameter. E.g.

```sh

#!/bin/bash
# shellcheck version=0.9.0 disable=SC2317

```
Maybe use a design that allows multiple versions and multiple checks

Use case is to disable SC2317 for now. See discussions here
https://github.com/koalaman/shellcheck/issues/2542
https://github.com/koalaman/shellcheck/issues/2613

But the check may improve in future. So we want to re-visit the issue with a future shellcheck version.

#### Here's what shellcheck currently says:

not possible

#### Here's what I wanted or expected to see:

The check is disabled, but only for the specific version of shellcheck
